OAR/curl/request_doi.py

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env python
+
+import json,cgi,time
+import httplib2, sys, base64, codecs
+
+res=[]
+retCode=0
+errCode=''
+doi='11623'
+res =  "%s/sci-gaia:%s" % (doi,time.time())
+print "Content-type: application/json\n\n"
+print json.dumps(res)
```

Comment:
@mtorrisi I take back what I said before, sorry. This is ok - I would just change doi -> prefix and res->doi